main/res/values/strings.xml

```diff
@@ -236,6 +236,7 @@
     <string name="info_stored_image">New image saved to:</string>
     <string name="info_storing_static_maps">Trying to store static maps</string>
     <string name="info_cache_saved">The cache has been stored locally</string>
+    <string name="err_trackable_log_not_anonymous">Logging trackables from c:geo cannot be anonymous. Please connect to the service.</string>
```

Comment:
GK could logged anonymously directly on geokrety.org. But as we use the API, authentication is mandatory. Users used to log anonymously on GK website may be disturbed.

What about "Logging trackables from c:geo cannot be done anonymously. Please fill your credentials in the %s settings"?